### ISSUE TITLE
Bug: Colors Contrast Issues

### ISSUE THREAD

**Issue by Iker1211** [TheOdinProject/theodinproject#4815]
### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] The title of this issue follows the `Bug: brief description of bug` format, e.g. `Bug: Lesson complete button does not update on click`
- [X] Would you like to work on this issue?

### Bug description

When open the CSS overview in Chrome Toolkit, in Colors, shows some contrast issues in the background-color, font-color. 
![issue](https://github.com/user-attachments/assets/380bf003-1f61-4142-835d-cf9e736c1b95)

### How to reproduce

1. Sign In
2. F12 to open DevToolKit 
3. Click on the Customize and control DevTools icon, side of the settings icon
4. Hover on more tools
5. Click on CSS overview
6. Click on capture overview
7. Click in Colors in the left panel of the window

### Expected behavior

The contrast between background-color and font-color in some of the elements pointed out in the CSS overview is not enough, especially when the font is small enough. The bug can also be seen in the Dark Mode and in the lessons route. 

### What browsers are you seeing the problem on?

Chrome

### What OS are you using?

Windows

### Additional Comments

Odin Project is pro

**Comment by KevinMulhern** [comment]
Thanks for reporting @Iker1211, I've assigned it to you! 

AA is the conformance level we're aiming for so we don't have too much to fix here. I think making the text a bit darker on any AA failures should satisfy this?

**Comment by KevinMulhern** [comment]
Hey @Iker1211, how are you getting on with this?

**Comment by ChuqiaoHuang** [comment]
Hi, I am willing to be assigned on this issue if possible.

**Comment by KevinMulhern** [comment]
Thanks for the interest @ChuqiaoHuang, do you understand what we're aiming for here? it's just the single AA violation we want fixed

**Comment by ChuqiaoHuang** [comment]
Yes, I understand! I’ll fix the single AA contrast issue. @KevinMulhern

**Comment by KevinMulhern** [comment]
Great stuff, thanks @ChuqiaoHuang, I've assigned it to you!

**Comment by ChuqiaoHuang** [comment]
@KevinMulhern I have fixed the AA contrast issue for the footer's <h2> element. However, I noticed there might also be contrast issues in the admin section. I don't have access to an admin account, so I couldn't fully check those pages. Do you need me to work on that as well?

**Comment by ChuqiaoHuang** [comment]
Hi @KevinMulhern  I’ve checked the admin account and confirmed that there are no AA contrast issues. Looks good to merge — apologies for the confusion earlier.

**Comment by KevinMulhern** [comment]
LL good, thanks again 💪
